nautobot/utilities/tests/test_filters.py

```diff
@@ -181,98 +182,111 @@ def test_filter_single_name(self):
         kwargs = {"power_panels": ["test-power-panel1"]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
 
-        self.assertCountEqual(list(qs), [self.site1])
+        self.assertQuerysetEqual(qs, [self.site1])
 
     def test_filter_single_pk(self):
 
         kwargs = {"power_panels": [self.power_panel1.pk]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
 
-        self.assertCountEqual(list(qs), [self.site1])
+        self.assertQuerysetEqual(qs, [self.site1])
 
     def test_filter_multiple_name(self):
 
         kwargs = {"power_panels": ["test-power-panel1", "test-power-panel2"]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
 
-        self.assertCountEqual(list(qs), [self.site1, self.site2])
+        self.assertQuerysetEqual(qs, [self.site1, self.site2])
 
     def test_filter_duplicate_name(self):
 
         kwargs = {"power_panels": ["test-power-panel3"]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
 
-        self.assertCountEqual(list(qs), [self.site1, self.site2])
+        self.assertQuerysetEqual(qs, [self.site1, self.site2])
 
     def test_filter_null(self):
 
         kwargs = {"power_panels": [settings.FILTERS_NULL_CHOICE_VALUE]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
+        expected_result = Site.objects.filter(powerpanel__isnull=True)
 
-        self.assertCountEqual(list(qs), [self.site0])
+        self.assertQuerysetEqual(qs, expected_result)
```

Comment:
not blocking, but would be good to use `assertQuerysetEqualAndNotEmpty` in cases like this.